core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -110,12 +167,12 @@ oppia.directive('audioControls', [
                 // immediately start playing the newly requested audio.
                 loadAndPlayAudioTranslation();
               }
-            }
+            } 
```

Comment:
Trailing spaces, here and below. If you're using sublime, you can add `"trim_trailing_white_space_on_save": true,` to your user prefs.

Reply:
done, thanks for the tip :)